doc/source/whatsnew/v0.21.0.txt

```diff
@@ -258,51 +270,71 @@ as in :meth:`DataFrame.rename`.
 
 .. warning::
 
-   To assist with upgrading pandas, ``rename_categories`` treats ``Series`` as
-   list-like. Typically, Series are considered to be dict-like (e.g. in
-   ``.rename``, ``.map``). In a future version of pandas ``rename_categories``
-   will change to treat them as dict-like. Follow the warning message's
-   recommendations for writing future-proof code.
-
-  .. ipython:: python
-     :okwarning:
+    To assist with upgrading pandas, ``rename_categories`` treats ``Series`` as
+    list-like. Typically, Series are considered to be dict-like (e.g. in
+    ``.rename``, ``.map``). In a future version of pandas ``rename_categories``
+    will change to treat them as dict-like. Follow the warning message's
+    recommendations for writing future-proof code.
 
-     c.rename_categories(pd.Series([0, 1], index=['a', 'c']))
+    .. code-block:: ipython
 
+        In [33]: c.rename_categories(pd.Series([0, 1], index=['a', 'c']))
+        FutureWarning: Treating Series 'new_categories' as a list-like and using the values.
+        In a future version, 'rename_categories' will treat Series like a dictionary.
+        For dict-like, use 'new_categories.to_dict()'
+        For list-like, use 'new_categories.values'.
+        Out[33]:
+        [0, 0, 1]
+        Categories (2, int64): [0, 1]
 
-See the :ref:`documentation <groupby.pipe>` for more.
 
 .. _whatsnew_0210.enhancements.other:
 
 Other Enhancements
 ^^^^^^^^^^^^^^^^^^
 
-- The ``validate`` argument for :func:`merge` now checks whether a merge is one-to-one, one-to-many, many-to-one, or many-to-many. If a merge is found to not be an example of specified merge type, an exception of type ``MergeError`` will be raised. For more, see :ref:`here <merging.validation>` (:issue:`16270`)
-- Added support for `PEP 518 <https://www.python.org/dev/peps/pep-0518/>`_ (``pyproject.toml``) to the build system (:issue:`16745`)
+New functions or methods:
```

Comment:
make these sub-sections

Reply:
checking locally how it looks (problem is that you hardly see the difference between the current section and subsection, and this would add a subsubsection, and don't want to do that if there is hardly a difference. I can eg make it bold though, if the subsubsection doesn't work out nicely)